databases/concessions_db.txt

```diff
@@ -1,8 +1 @@
-Kids Combo, 10e
-Popcorn, 7e
-Drink, 4e
-Hot Dog, 7e
-Nachos, 6e
-Chocolate Bites, 4e
-Twix, 7e
-Pizza, 15e
+popcorn,hotdog,cola,nachos,
```

Comment:
I removed nachos from this database, then logged in as admin. I could not add the nacho concession back. "All concessions have been added" even though only three in the selector.